pyworkflow/object.py

```diff
@@ -798,9 +798,33 @@ def __nonzero__(self):
         return self.get() 
     
     def __bool__(self):
-        return self.get()  
-    
-    
+        return self.get()
+
+
+class Range(String):
```

Comment:
I don't see the need for this new class

Reply:
Wouldn't this turn user input "1,5-8,10" into [1,5,6,7,8,10] in the form next time it loads the protocol?